lib/server.js

```diff
@@ -132,15 +133,23 @@ function createServer(options, clients, name) {
         name: common.METRIC_REQUEST_COUNTER,
         help: 'count of Muskie requests completed'
     });
-    // A histogram to track the time to first byte.
+    /*
+     * A log-linear histogram to track the time to first byte.
+     * Track values between 10^(-4) and 10^(4) seconds.
+     */
     options.collector.histogram({
         name: common.METRIC_LATENCY_HISTOGRAM,
-        help: 'time-to-first-byte of Muskie requests'
+        help: 'time-to-first-byte of Muskie requests',
+        buckets: artedi.logLinearBuckets(10, -4, 3, 4)
```

Comment:
There is no particular reason these values have been chosen. AFAIK, we don't have requests that take thousands of seconds for the first byte to be served to the user. However, we could have requests that take 1000s of seconds if the user is downloading a large object, and I wanted to use the same number of buckets for the two histograms. I am open to change that if you think this is an issue.

Reply:
My main concern here is that having more buckets in the histogram increases the amount of data stored and the processing required. We don't currently have guidelines but I just wanted to make sure this is something that had been thought through so that we have a record of the reasoning.

You said you wanted to use the same number for the two, is there some reasoning behind that?

Reply:
I understand your concern and I will update the code to reduce the number of buckets. There is no reason for having the same number of buckets.